mitodl/open-discussions
# Added comments REST API

#### What are the relevant tickets?
Fixes #61 

#### What's this PR do?
Adds a REST API for comments

#### How should this be manually tested?

 - Go to `/api/v0/posts/{post_id}/comments/` and create a new comment for your post
 - Create a new comment replying to the comment you just created
 - Refresh `/api/v0/posts/{post_id}/comments/` to verify the listing of comments includes the ones you created
 - Go to `/api/v0/comments/{comment_id}/` to verify that you can see your comment
 - In that form edit a comment by either changing the `upvoted` value or the `text` value via PATCH.
 - Delete the comment
